INSTALLATION.md

```
@@ -380,3 +390,84 @@ If you don't want this hook to run, you can manually opt out of this using the `
+   sudo chown -R talawa_admin:talawa_admin /path/to/talawa-admin
+   sudo chmod 750 /path/to/talawa-admin
+   sudo find /path/to/talawa-admin -type f -exec chmod 640 {} \
+   sudo chmod 600 /path/to/talawa-admin/.env
+   ```
+
+---
+
+### Service Configuration
+
+#### 1. **Copy the `talawa_admin.service` file**
+   - Place the `talawa_admin.service` file in the appropriate systemd directory based on your Linux distribution:
```

Markdownlint (0.37.0)

Expected: 0; Actual: 3
Unordered list indentation

(MD007, ul-indent)